doc/user-guide.md

```diff
@@ -156,7 +156,15 @@ return reconcile.Result{}, err
 return reconcile.Result{Requeue: true}, nil
 ```
 
-**// TODO:** Doc on controller-runtime client and examples on how to use it.
+You can set the `Result.RequeueAfter` to requeue the `Request` after a grace period as well:
+```Go
+import "time"
+
+// Reconcile for any reason than error after 5 seconds
+return reconcile.Result{RequeueAfter: time.Second*5}, nil
```

Comment:
This should actually be `time.Duration(5)*time.Second` otherwise there's a type mismatch. time.Second is int64.

Reply:
Seems to run fine as-is: https://play.golang.org/p/jex0ymMFT9A